particle-accelerator/.gitignore

```diff
@@ -0,0 +1,29 @@
+# Logs
+logs
+*.log
+
+# Runtime data
+pids
+*.pid
+*.seed
+
+# Coverage directory used by tools like istanbul
+coverage
+.eslintcache
+
+# Dependency directory
+# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
+node_modules
+
+# OSX
+.DS_Store
+
+release/app/dist
+release/build
+.erb/dll
+
+.idea
```

Comment:
Looks like they tried to ignore the `.idea` directory. So maybe these values just need to be moved into the `.gitignore` in the root of the repo. Same for the `.DS_Store` file.

Reply:
Okay (done)